terra_notebook_utils/utils.py

```diff
@@ -32,3 +32,14 @@ def __exit__(self, *args, **kwargs):
                 pass
             self.prune_futures()
         self._executor.shutdown()
+
+def concurrent_recursion(process_item: Callable[[Any], Iterable[Any]], items: Iterable[Any], concurrency: int=8):
```

Comment:
Nit: ` -> None`

Reply:
I’ve never used this anywhere and don’t intend to start.